src/platform/testing/e2e/cypress/support/commands/expandAccordions.js

```diff
@@ -1,15 +1,35 @@
 /**
  * Expands all accordions and AdditionalInfo components.
+ * Web Components that require Shadow DOM broken out from React Components
  */
 Cypress.Commands.add('expandAccordions', () => {
   Cypress.log();
 
   cy.get('main').then($main => {
-    const accordions = $main.find('button[aria-expanded=false]');
-    if (accordions.length) {
-      cy.wrap(accordions).each(button => {
-        cy.wrap(button).click({ force: true });
-      });
+    if ($main.find('va-accordion-item').length > 0) {
+      // va-accordion-item Web Component Expand
+      cy.get('va-accordion-item')
+        .shadow()
+        .find('button[aria-expanded=false]')
+        .each(button => {
+          cy.wrap(button).click({ force: true });
+        });
+    } else if ($main.find('va-additional-info').length > 0) {
+      // va-additional-info Web Component Expand
+      cy.get('va-additional-info')
+        .shadow()
+        .find('a[role="button"][aria-expanded=false]')
+        .each(button => {
+          cy.wrap(button).click({ force: true });
+        });
+    } else {
+      // AdditionalInfo and Accordion React Components Expand
+      const accordions = $main.find('button[aria-expanded=false]');
+      if (accordions.length) {
+        cy.wrap(accordions).each(button => {
+          cy.wrap(button).click({ force: true });
+        });
+      }
```

Comment:
The way I'm understanding this, having this command open `<AdditionalInfo>` seems like a side-effect rather than intended behavior. If that's behavior that we want to explicitly include then that's fine, but maybe the name should be accordion-specific.

More importantly, do we want these three cases to be mutually exclusive? For example, it looks like any `<va-additional-info>` components on the page would prevent the traditional accordions from being expanded, and any `<va-accordion-item>`s would prevent the additional info components from expanding.